content/participants/ttjc@yashmantri20/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Yash Mantri
+type: "participant"
+name: "Yash Mantri"
+portfolio: "https://yashmantri20.github.io/yashmantri/"
+github-link: "https://github.com/yashmantri20"
+linkedin-link: "https://www.linkedin.com/in/yashmantri20/"
+looking-for: "internship"
+description: "HTML | CSS | JavaScript | ReactJs | GraphQL | NodeJs | MongoDB"
+---
+
+Hi! I am Yash Mantri let me walk you through my intro, projects.
+
+### introduction
+
+I am a student from Computer Science branch from Uka Tarsadia University, Bardoli.
+Currently, I am in third year and I love coding. 
+I am eager to learn new things and apply them in real world problems.
+
+## why are you taking this challenge?
+
+I want to enhance my coding skills and build projects.
+These projects would enhance my skills and it will be helpful to build my portfolio strong.
+Plus I need the internship.
+
+## something interesting about you?
+
+Nothing much !!
+In 12th when the result was out I was not satisfied with my result and then I was confused should I select Science or not but at the end I selected science and I'm here.
+
+### help I need?
+
+I need help in the backend.
+
+### how to contact me?
+
+- [Twitter](https://twitter.com/yashmantri20)
+- [Telegram](https://t.me/yashmantri)
+
+### projects
+
+There are not much but still there are three that I made for Hackathons and Competitions.
+
+My projects:
+
+#### Dectro : Swachh Bin
+
+_description_ In this application there are two modules :- 1) Users : Through this app the Shepherds and Farmers can order vegetable waste , organic waste and animal waste , OTP verification . 2) Admin : The admin can see the total number of orders , complains which are stored in database. When the admin accepts the order the customer will receive the message regarding the arrival date of the order .
+
+_stack_ Made using Android Studio , Firebase for database.
+
+_hosted link_ Not Hosted
+
+_github link_ https://github.com/yashmantri20/Swachh-Bin
+
+#### Dholpur : Covid19 Traker 
+
+_description_ The tracker gives the information regarding total positive cases , total active cases , total recovered , total deaths , it also includes a self assessment test to check whether you are safe or not and gives information regarding symptoms , testing and state wise data .
+
+_stack_ ReactJs , NodeJs , HTML and CSS.
+
+_hosted Link_ https://covid19-india-app.netlify.com
+
+_github link_ https://github.com/yashmantri20/covid-19
+
+#### Musicee : Motion Detection
```

Comment:
Hey! @yashmantri20 this project was done before the start of jobChallenge. Only projects made during the challenge are accepted. Please remove it.

Reply:
I have to remove motion detection project right ??

Reply:
I have removed that project